exporter/loggingexporter/config.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package loggingexporter
+
+import (
+	"github.com/open-telemetry/opentelemetry-service/models"
+)
+
+// ConfigV2 defines configuration for logging exporter.
+type ConfigV2 struct {
+	models.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
```

Comment:
Nit: consider adding a TODO to allow configure logger levels.

Reply:
Done.